Stripe/STPPaymentMethodParams.m

```diff
@@ -107,4 +108,53 @@ + (nonnull NSDictionary *)propertyNamesToFormFieldNamesMapping {
              };
 }
 
+#pragma mark - STPPaymentOption
+
+- (UIImage *)image {
+    if (self.type == STPPaymentMethodTypeCard && self.card != nil) {
+        STPCardBrand brand = [STPCardValidator brandForNumber:self.card.number];
+        return [STPImageLibrary brandImageForCardBrand:brand];
+    } else {
+        return [STPImageLibrary brandImageForCardBrand:STPCardBrandUnknown];
+    }}
+
+- (UIImage *)templateImage {
+    if (self.type == STPPaymentMethodTypeCard && self.card != nil) {
+        STPCardBrand brand = [STPCardValidator brandForNumber:self.card.number];
+        return [STPImageLibrary templatedBrandImageForCardBrand:brand];
+    } else if (self.type == STPPaymentMethodTypeFPX) {
+        return [STPImageLibrary bankIcon];
+    } else {
+        return [STPImageLibrary templatedBrandImageForCardBrand:STPCardBrandUnknown];
+    }
+}
+
+- (NSString *)label {
+    switch (self.type) {
+        case STPPaymentMethodTypeCard:
+            if (self.card != nil) {
+                STPCardBrand brand = [STPCardValidator brandForNumber:self.card.number];
+                NSString *brandString = STPStringFromCardBrand(brand);
+                return [NSString stringWithFormat:@"%@ %@", brandString, self.card.last4];
+            } else {
+                return STPStringFromCardBrand(STPCardBrandUnknown);
+            }
+        case STPPaymentMethodTypeiDEAL:
+            return @"iDEAL";
```

Comment:
hmm, we localized apm names for sources but not payment methods (see STPSource.m). Probably this and FPX should be wrapped in `STPLocalizedString`

Reply:
looks like we're also not localizing in STPPaymentMethod.m

Reply:
Yeah, it looks like localization was largely dropped for PaymentMethod-related stuff. We need to audit more of it.

Reply:
presumably these are all the same as the localized values in `STPSource` so we should be able to just add the `STPLocalizedString` and then get translations (if there are any) for FPX. Can do as a followup